nontclient/src/Router/mainRouter.js

```diff
@@ -10,6 +10,8 @@ import ReserveInfo from "../Components/Reservation/ReserveInfo";
 import GuardedRoute from "./GuardedRoute";
 import NontOwnerRouter from "./NontOwnerRouter";
 import NontSitterRouter from "./NontSitterRouter";
+import AdminLogin from "../Components/Admin/AdminLogin";
+import AdminCreate from "../Components/Admin/AdminCreate";
```

Comment:
It ran perfectly after I had deleted this line.

Reply:
oh thanks, I forgot to remove it.